soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -765,6 +770,26 @@ void CreateDampesDiaryText() {
   dampesText = temp1 + area + temp2;
 }
 
+void CreateGregRupeeHint() {
+  uint32_t item = GREG_RUPEE;
+  uint32_t location = FilterFromPool(allLocations, [item](const uint32_t loc){return Location(loc)->GetPlaceduint32_t() == item;})[0];
+  Text area = GetHintRegion(Location(location)->GetParentRegionKey())->GetHint().GetText();
+
+  Text temp1 = Text{
+    "By the way, if you're interested, I saw the shiniest %gGreen Rupee%w somewhere in%g",
+    "",
+    ""
+  };
+
+  Text temp2 = {
+    "%w. It's said to have %rmysterious powers%w...^But then, it could just be another regular rupee.&Oh well.",
```

Comment:
```suggestion
    "%w. It's said to have %rmysterious powers%w...^But then, it could just be another regular rupee.&Oh well...",
```

Reply:
To be honest, I think the single period fits better than the ellipsis. This change seems subjectively mostly.

Reply:
yeah, i'm fine either way. i think i'm just more used to seeing "oh well..." than "oh well."